ambari-server/src/main/java/org/apache/ambari/server/topology/ClusterTopology.java

```diff
@@ -135,9 +135,9 @@
   Collection<String> getHostAssignmentsForComponent(String component);
 
   /**
-   * Get all of the services represented in the blueprint.
+   * Get all of the service types represented in the blueprint.
    *
-   * @return collection of all represented service names
+   * @return collection of all represented service types
    */
   Collection<String> getServices();
```

Comment:
Should we consider changing the name of this method as well, to make it clear that service types are now being returned from this method, rather than service names?

Reply:
Good point, renamed to `getServiceTypes`.